pfrl/collections/prioritized.py

```diff
@@ -222,7 +258,7 @@ def _find(index_left, index_right, node, pos):
             return _find(index_center, index_right, node_right, pos - left_value)
```

Comment:
Question: Is `pos` guaranteed to be a number? Here I see the comparator `pos < left_value` and `pos - left_value` seem to imply it's a number.

Reply:
Yeah, it is reasonable to assume `pos` is a Number because `left_value = 0.0`, thanks. I changed it to Number.